src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -947,4 +947,22 @@ bool IsCommodore(void* log)
     FREE_MEMORY(textResult);
 
     return status;
-}
\ No newline at end of file
+}
+
+enum SelinuxState {
+  SelinuxUnknown = 0,
+  SelinuxFound,
+  SelinuxNotFound,
```

Comment:
Unknown -1, found 0 and not found 1 would better match overall status style? 0 being true/success, etc.

Reply:
dropped the enum due to a later comment ("The SelinuxState can be simplified to a Boolean")